src/docs/reference/modules/Email.Azure/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Email (`OrchardCore.Email.Azure`)
+
+This module provides the infrastructure necessary to send emails using `Azure Communication Services`.
+
+## Azure Email Settings
```

Comment:
This section is outdated I think, because the we only show the default sender and whether the connection string is configured or not.